### PR TITLE
Use kapprover package to query for pod SANs

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: deployments/tls-app.yaml [hunk 1]
```diff
@@ -24,17 +24,10 @@ spec:
...
-                fieldRef:
-                  fieldPath: status.podIP
           args:
-            - "-additional-dnsnames=example.com"
-            - "-cert-dir=/etc/tls"
             - "-namespace=$(NAMESPACE)"
-            - "-pod-ip=$(POD_IP)"
             - "-pod-name=$(POD_NAME)"
-            - "-service-names=tls-app"            
+            - "-query-k8s"
```
Comment (raghu67) [inline deployments/tls-app.yaml:30]: Why control this based on a command line flag? Why not always query?

**Review by raghu67 — COMMENTED** [review]
Why control the query of pod SANs with a flag? Why not always query?

File: deployments/tls-app.yaml (hunk 1 above)
Comment (johngmyers) [reply deployments/tls-app.yaml:30]: I'm in contact with someone else who wants to use this project for client certificate use cases, where these semantics wouldn't be wanted.

**Review by raghu67 — APPROVED** [review]
LGTM

**Pull request merged**